tests/package/Makefile

```diff
@@ -5,56 +5,47 @@ default: all
-
-debian-systemd/mesos-version: target/mesos-version
-	if [ $$(cat target/mesos-version) = "1.5.0-2.0.1" ]; then echo "1.5.0-2.0.2" > $@; else echo "Package version patch should not be needed; remove this step from the Makefile"; exit 1; fi
-
-ubuntu-upstart/mesos-version: target/mesos-version
-	if [ $$(cat target/mesos-version) = "1.5.0-2.0.1" ]; then echo "1.5.0-2.0.2" > $@; else echo "Package version patch should not be needed; remove this step from the Makefile"; exit 1; fi
-
 target/mesos-version: ../../project/Dependencies.scala
 	mkdir -p target
 	cat ../../project/Dependencies.scala | grep MesosDebian | cut -f 2 -d '"' > $@.tmp
 	[ "$$(wc -l $@.tmp | awk '{print $$1}')" = 1 ]
 	mv $@.tmp $@
 
-target/debian-systemd.built: debian-systemd/Dockerfile debian-systemd/mesos-version
-	cd debian-systemd && docker build . -t marathon-package-test:debian-systemd
+target/debian8.built: debian8/Dockerfile debian8/mesos-version
+	cd debian8 && docker build . -t marathon-package-test:debian8
 	mkdir -p target
 	touch $@
 
-target/debian-systemv.built: target/debian-systemd.built debian-systemv/Dockerfile
-	cd debian-systemv && docker build . -t marathon-package-test:debian-systemv
+target/ubuntu1404.built: ubuntu1404/Dockerfile ubuntu1404/mesos-version
+	cd ubuntu1404 && docker build . -t marathon-package-test:ubuntu1404
+	mkdir -p target
 	touch $@
 
-target/mesos.built: target/debian-systemd.built mesos/Dockerfile
-	cd mesos && docker build . -t marathon-package-test:mesos
+target/ubuntu1604.built: ubuntu1604/Dockerfile ubuntu1604/mesos-version
+	cd ubuntu1604 && docker build . -t marathon-package-test:ubuntu1604
+	mkdir -p target
 	touch $@
 
-target/centos-systemv.built: centos-systemv/Dockerfile centos-systemv/mesos-version
-	cd centos-systemv && docker build . -t marathon-package-test:centos-systemv
+target/centos6.built: centos6/Dockerfile centos6/mesos-version
+	cd centos6 && docker build . -t marathon-package-test:centos6
 	touch $@
 
-target/centos-systemd.built: centos-systemd/Dockerfile centos-systemd/mesos-version
-	cd centos-systemd && docker build . -t marathon-package-test:centos-systemd
+target/centos7.built: centos7/Dockerfile centos7/mesos-version
+	cd centos-systemd && docker build . -t marathon-package-test:centos7
 	touch $@
 
-target/ubuntu-upstart.built: ubuntu-upstart/Dockerfile ubuntu-upstart/mesos-version
-	cd ubuntu-upstart && docker build . -t marathon-package-test:ubuntu-upstart
+target/mesos.built: target/debian8.built mesos/Dockerfile
+	cd mesos && docker build . -t marathon-package-test:mesos
 	touch $@
 
 test: | all
 	amm test.sc all
 
-all: target/mesos.built target/debian-systemv.built target/debian-systemd.built target/centos-systemd.built target/centos-systemv.built target/ubuntu-upstart.built
+all: target/mesos.built target/debian8.built target/centos7.built target/centos6.built
```

Comment:
seems like make all should include ubuntu1404 and 1604?

Reply:
Good catch.